### ISSUE TITLE
Need a PV for NFS

### ISSUE THREAD

**Issue by John-Lin** [linkernetworks/vortex#246]
We'll have a PV for NFS and it can be bounded with the mongodb statefulset.

The YAML place in `deploy/kubernetes` folder like
```bash
~/go/src/github.com/linkernetworks/vortex/deploy/kubernetes [git::develop] [johnlin@Ches-MacBook-Pro] [0:14]
> tree base
base
|-- hostpath
|   `-- manual.yaml
`-- nfs
    `-- nfs-pv.yaml

3 directories, 1 file
```

The example PV

```yaml
apiVersion: v1
kind: PersistentVolume
metadata:
  name: mongo-storage 
spec:
  capacity:
    storage: 1Gi 
  accessModes:
  - ReadWriteMany
  nfs: 
    path: /nfsshare 
    server: 10.14.1.100
  persistentVolumeReclaimPolicy: Recycle 
```
We need a PR for making helm YAML and make sure `spec.capacity.storage`, `spec.accessModes`, `spec.nfs.path`, `spec.nfs.server` are configurable

**Comment by John-Lin** [comment]
For the NFS storage configs, we should put it in `deploy/helm/config/production.yaml` and create makefile targets for launching production and development environment separately.

**Comment by hwchiu** [comment]
就把 pash/server 的參數拉出來就好?
這樣是不是要 ansible要去安裝 NFS ?

**Comment by John-Lin** [comment]
恩, 因為我們NFS用來測試 可以考慮不用放到ansible

**Comment by John-Lin** [comment]
#247

**Comment by John-Lin** [comment]
closed #247
